### PR TITLE
Rename ValuesQuerySet -> _ValuesQuerySet and remove _BaseQuerySet. Ma…

### PR DESCRIPTION
**Pull request by syastrov**
…ke a public alias called ValuesQuerySet in django_stubs_ext.

This is currently a PoC to see if tests pass.

# I have made things!

### CODE REVIEW

File: django-stubs/db/models/query.pyi [hunk 1]
```diff
@@ -155,13 +156,14 @@ class NamedValuesListIterable(ValuesListIterable): ...
...
-class ValuesQuerySet(_BaseQuerySet[_T], Collection[_Row], Sized):
+class _ValuesQuerySet(Generic[_T, _Row], Collection[_Row], Reversible[_Row], QuerySet[_T], Sized): # type: ignore
+    def __len__(self) -> int: ...
     def __contains__(self, x: object) -> bool: ...
     def __iter__(self) -> Iterator[_Row]: ...  # type: ignore
     @overload  # type: ignore
     def __getitem__(self, i: int) -> _Row: ...
     @overload
-    def __getitem__(self: _QS, s: slice) -> _QS: ...
+    def __getitem__(self: _QS, s: slice) -> _QS: ... # type: ignore
```
Comment (syastrov) [inline django-stubs/db/models/query.pyi:165]: It was necessary to silence some error about self type being required for non-static-method. I am not sure the cause, but perhaps it was related to the error about MRO.

**Review by sobolevn — APPROVED** [review]
LGTM! Thank you!

But, tests are failing, can you please fix it?

**Review by sobolevn — APPROVED** [review]
Thanks again!

**Comment by sobolevn** [comment]
@syastrov would you like to join our org? You make so much impact! I have sent you an invite!

**Comment by syastrov** [comment]
Sure, thanks! I've joined

**Pull request merged**